source/documentation/deploying_services/postgresql.md.erb

```diff
@@ -244,6 +246,61 @@ To move data between two PaaS-hosted PostgreSQL databases:
 
     where `DESTINATION_SERVICE_NAME` is the name of the target database.
 
+### Expected Errors
+
+#### The errors that are expected from both types of import operation above are:
```

Comment:
This does not need to be a heading. 
I _believe_ we try to avoid referring to parts of the text by using directions like "above" as it makes it less accessible and hard for users to find (I might be making this up)
```suggestion
When importing data into a target database you may encounter errors which do not stop the import process from finishing:
```